.eslintrc.json

```diff
@@ -1,24 +1,27 @@
 {
-  "env": {
-    "node": true,
-    "commonjs": true,
-    "es2017": true,
-    "mocha": true
-  },
-  "ignorePatterns": ["**/*.mjs", "**/*.js"],
+  "ignorePatterns": ["build", "coverage"],
   "extends": [
     "eslint:recommended",
     "plugin:@typescript-eslint/recommended-type-checked",
     "plugin:@typescript-eslint/stylistic-type-checked",
+    "plugin:import/recommended",
+    "plugin:import/typescript",
```

Comment:
The `recommended` and `typescript` configurations are now included for the `import` plugin.